hathor/manager.py

```diff
@@ -619,7 +631,8 @@ def _initialize_components_new(self) -> None:
             sys.exit()
 
         # restart all validations possible
-        self._resume_validations()
+        if self._enable_sync_v2:
+            self._resume_validations()
```

Comment:
Should we rename it to `_sync_v2_resume_validation(...)`?

Reply:
That makes sense.

Reply:
Done.

Reply:
Thanks!